modules/__tests__/enhancer-test.js

```diff
@@ -3,7 +3,8 @@ var Enhancer = require('inject!enhancer.js')({
   './resolve-styles.js': resolveStyles
 });
 
-var {Component} = require('react');
+var React = require('react/addons');
```

Comment:
After some crude testing it doesn't look like it makes much of a difference (if it does, it's very marginal), but we should do this for consistency's sake anyway - so here it is: https://github.com/FormidableLabs/radium/pull/312